loadtest/control/simulcontroller/actions.go

```diff
@@ -870,3 +870,101 @@ func searchChannels(u user.User) control.UserActionResponse {
 		return control.UserActionResponse{Info: fmt.Sprintf("found %d channels", len(channels))}
 	})
 }
+
+func searchPosts(u user.User) control.UserActionResponse {
+	team, err := u.Store().CurrentTeam()
+	if err != nil {
+		return control.UserActionResponse{Err: control.NewUserError(err)}
+	} else if team == nil {
+		return control.UserActionResponse{Err: control.NewUserError(errors.New("current team should be set"))}
+	}
+
+	var words []string
+	var opts control.PostsSearchOpts
+	// This is an arbitrary limit on the number of words to search for.
+	// TODO: possibly use user analytics data to improve this.
+	count := 1 + rand.Intn(4)
+
+	// TODO: back the probability of these choices with real data.
```

Comment:
Let's file a ticket for this? Now that you have access, it shouldn't be very hard to get this.

Reply:
I am not sure we collect this kind of data. To improve this we would need to store the actual text content of what the users are searching.
Were you thinking of something else?

Reply:
Oh yeah you're right. Nvm then.